### DISCUSSION TITLE
why osu!lazer might not feel like standard osu (in my opnion)

### DISCUSSION THREAD

**Discussion by Arvesh26** [ppy/osu#23120] [Q&A]
### Type

Suggestion

### Description

firstly i think that the hit circles font number is a bit larger compared to standard. 
the sliders slider head fades in with the slider ball when hit a little early.

also the group by difficulty is not same as the standard version. it is a subtle change but it makes finding maps harder.

### Version

osu! 2023.326.1 - lazer

### Logs

[input.log](https://github.com/ppy/osu/files/11137696/input.log)
[network.log](https://github.com/ppy/osu/files/11137697/network.log)
[performance.log](https://github.com/ppy/osu/files/11137698/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11137699/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11137700/updater.log)
[database.log](https://github.com/ppy/osu/files/11137701/database.log)

**Comment by Rekunan** [comment]
Grouping by difficulty is a known issue: #2547 #4919

For the first two, do you think you can attach some screenshots or videos comparing the two? It'd help with code direction and to confirm if they really are as you say.

**Comment by Arvesh26** [comment]
sure, here you go
oh and btw turns out the font size of the hit circle was same, it felt different cuz i changed my skin

here are the videos
this is slider in standard: https://youtu.be/npLNGc7o05w
this is slider in lazer: https://youtu.be/3KtepCrCVw8

**Comment by Rekunan** [comment]
From what I'm seeing, you're talking about the slider head following the slider track. This is an intentional change/difference from stable and is listed in the GitHub wiki page: https://github.com/ppy/osu/wiki/Gameplay-differences-from-osu!stable#slider-head-circle-follows-track-snaking-when-not-hit
